vwr/driver.py

```diff
@@ -83,10 +86,14 @@ def turn_off(self):
 
     def get(self):
         """Gets the setpoint and internal temperature."""
-        return {'setpoint': self.get_setpoint(),
-                'actual': self.get_internal_temperature(),
-                'pump': self.get_pump_speed(),
-                'on': self.get_operating_status()}
+        response = {'setpoint': self.get_setpoint(),
+                    'actual': self.get_internal_temperature(),
+                    'pump': self.get_pump_speed(),
+                    'on': self.get_operating_status()}
+        if self.connected:
+            return response
+        else:
+            return {'connected': False}
```

Comment:
If you're going to return `connected: False` when disconnected, you should return `connected: True` when connected.

More generally, why even try to get data if not connected? How about:

```python
if self.connected:
    return {'setpoint': self.get_setpoint(), ..., 'connected': True}
else:
    return {'connected': False}
```